bedrock/releasenotes/tests/test_models.py

```diff
@@ -43,12 +43,15 @@ def setUp(self):
 
     def _add_in_ff100(self):
         # TODO: remove once FF100 data is properly in the dataset
-        last = models.ProductRelease.objects.filter(channel="Release").last()
+        last = models.ProductRelease.objects.filter(channel="Nightly").last()
```

Comment:
@stevejalim This seems to have been the important bit since we are querying for a nightly version string below.

Reply:
Thanks! Makes me wonder how the test passed at all on a local build...